docs/docs-content/profiles/cluster-profiles/create-cluster-profiles/create-full-profile.md

```diff
@@ -34,12 +34,16 @@ Create a full profile by first adding infrastructure layers composed of an Opera
 
   To learn how to create multiple profile versions that use the same name, check out [Version a Cluster Profile](../modify-cluster-profiles/version-cluster-profile.md).
 
-5. Select the infrastructure provider, managed Kubernetes, or technical preview cloud type for your environment and click **Next**.
+5. Select the Infrastructure Provider, Managed Kubernetes, or Tech Preview cloud type for your environment and click **Next**.
 
   :::info
+  Nutanix is a Tech Preview cloud type intended for Nutanix clusters deployed using Palette's generic framework built upon the open-source Cluster API (CAPI) initiative.
 
-  Cluster profiles created from cloud types that are under technical preview are intended for clusters that a cloud provider deploys using Palette's generic framework built upon the open-source Cluster API (CAPI) initiative. When creating this type of profile, you do not have to specify anything for the OS or Kubernetes layers. Out-of-the-box packs are provided for the network and storage profile layers. Additionally, Palette provides a cloud provider-specific storage pack. 
-  
+  When creating a Nutanix profile, you do not have to specify anything for the OS or Kubernetes layers. Palette provides out-of-the-box packs for the network and storage profile layers, including the **Nutanix CSI** storage pack.
+
+  <!-- <<< Save this text for future when more clouds may be listed under Tech Preview:>>> Cluster profiles created from a Tech Preview cloud type are intended for clusters that a cloud provider deploys using Palette's generic framework built upon the open-source Cluster API (CAPI) initiative.  
```

Comment:
I think we're better off using the generic text below. It's awkward that we call out Nutanix specifically. I rather we use the text below.

Reply:
I agree. Revised. ✅